mediacapture-streams/stream-api/mediastream/mediastream-idl.html

```diff
@@ -3,12 +3,12 @@
 <head>
 <title>MediaStream constructor algorithm</title>
 <link rel="author" title="Dominique Hazael-Massieux" href="mailto:dom@w3.org"/>
-<link rel="help" href="http://dev.w3.org/2011/webrtc/editor/getusermedia.html#idl-def-MediaStream">
-<link rel="help" href="http://dev.w3.org/2011/webrtc/editor/getusermedia.html#widl-MediaStream-id">
-<link rel="help" href="http://dev.w3.org/2011/webrtc/editor/getusermedia.html#mediastream">
-<link rel="help" href="http://dev.w3.org/2011/webrtc/editor/getusermedia.html#event-mediastream-ended">
-<link rel="help" href="http://dev.w3.org/2011/webrtc/editor/getusermedia.html#widl-MediaStreamTrack-stop-void">
-<link rel="help" href="http://dev.w3.org/2011/webrtc/editor/getusermedia.html#widl-MediaStreamTrack-clone-MediaStreamTrack">
+<link rel="help" href="http://w3c.github.io/mediacapture-main/getusermedia.html#idl-def-MediaStream">
```

Comment:
Is referring to the github version of the drafts (the ED drafts) kosher in tests? Or should we continue to prefer the WD drafts?